changelogs/fragments/801-fix-delete-without-ttl.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

```suggestion
minor_changes:
```

I'd either call this a change: it's not really a bugfix since it was documented behaviour.